modal/functions.py

```diff
@@ -1511,4 +1512,29 @@ async def _gather(*function_calls: _FunctionCall):
         raise exc
 
 
+class _GroupedFunction(typing.Generic[P, R], _Object, type_prefix="gfu"):
```

Comment:
I'm not totally sure it's legal to have a three-letter type prefix ... what if this was `FunctionGroup` and the prefix was `fg-`?

Reply:
Yeah it's not